### PR TITLE
CLOUDP-201386 Update the deployment start|pause command to add a spinner

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Update the deployment start|pause command to add a spinner

_Jira ticket:_ CLOUDP-201386

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/deployments/start.go [hunk 1]
```diff
@@ -91,6 +91,9 @@ func (opts *StartOpts) RunLocal(ctx context.Context) error {
 }
 
 func (opts *StartOpts) startContainer(ctx context.Context, deployment options.Deployment) error {
+	opts.StartSpinner()
+	defer opts.StopSpinner()
```
Comment (tibulca) [inline internal/cli/atlas/deployments/start.go:95]: I think the second requirement was not too clear, but similar to how `setup` cmd works, the start cmd should wait for mongot to be running.
`update the deployment start command to wait for the containers to start before returning a successful message to the user`

Comment (fmenezes) [reply internal/cli/atlas/deployments/start.go:95]: oh I see, so we wait for mongot to be available?

Comment (fmenezes) [reply internal/cli/atlas/deployments/start.go:95]: I think pause is almost synchronous as podman sends signals to close the process

**Review by tibulca — APPROVED** [review]
LGTM, thanks for the mongot updates

**Review by andreaangiolillo — APPROVED** [review]

**Pull request merged**